### ISSUE TITLE
Display word Word Count while writing

### ISSUE THREAD

**Issue by binyamin** [forem/forem#5932]
It would be nice to have a word count displayed while the user is writing/editing an article. I feel that this would improve the entire dev.to experience, because it would keep people on dev.to for longer. I got the idea from [200wordsaday](https://cowriters.app), now *cowriters.app*.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: please wait until we add a `ready for dev` label to the issue before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @maestromac or @rhymes or @citizen428 from the DEV team and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by b3u** [comment]
@rhymes Thanks so much. Replies like these make contributing to open-source worthwhile.
Also, I'd love to work on this, but I'm not familiar with the codebase. Do you have a vague idea of where the code would go?

**Comment by rhymes** [comment]
Hi @b3u, thanks for the kind words!

As it's technically not approved yet I would suggest to come up with a proposal for the UI that can then be discussed before diving in into the editor's code.

What do you think?

**Comment by b3u** [comment]
>  I would suggest to come up with a proposal for the UI

How about right underneath the text editor? MS Word, LibreOffice, and Grammarly all place the word count there, IIRC. Maybe a bit lighter than the attached picture, indented from the left margin and in a font-size meant for details.

![dev-ftr1](https://user-images.githubusercontent.com/39805353/73966658-715ec100-48e4-11ea-8af1-89a63e9ec35e.png)

**Comment by citizen428** [comment]
This is nice and unobstrusive @b3u, I think it would make a nice edition. 👍

**Comment by sahaldos1** [comment]
Hi Team, is it okay if I take a look at this issue?

**Comment by citizen428** [comment]
@sahaldos1 Thanks for volunteering, I've assigned you the issue now :smiley:

**Comment by djamaile** [comment]
@sahaldos1 are you still working on this? Else I can take it.

**Comment by b3u** [comment]
@djamaile there's a PR waiting for approval. See here: https://github.com/forem/forem/pull/9421#issuecomment-662952559

**Comment by sahaldos1** [comment]
@djamaile like @b3u said, I actually finished it and submitted a pull request. It’s just pending approval.

**Comment by djamaile** [comment]
Ah sorry, good luck ❤️ @sahaldos1

**Comment by ludwiczakpawel** [comment]
https://github.com/forem/forem/pull/9421#issuecomment-731064580
